spec/helpers/search_helper_spec.rb

```diff
@@ -1,6 +1,68 @@
 require 'rails_helper'
 
 RSpec.describe SearchHelper, type: :helper do
+
+  describe "#paginate" do
+
+    let (:page_params) {{
+      :total_pages=>3,
+      :previous_link=>"/petitions?page=1&state=all",
+      :next_link=>"/petitions?state=all",
+      "model" => Petition
+    }}
```

Comment:
Also try to use modern hash syntax, e.g. `{ foo: "bar" }` over `{ :foo => "bar" }`